### PR TITLE
Fix publish properties shortcut

### PR DESCRIPTION
**Pull request by kickster97**
- More property tags are added to shortcut 
- Avoids adding a single tag multiple times 
- Adds payload encoding selector, you can now choose Base64 as well as String

(img: hovering over "Priority" tag)
image

### CODE REVIEW

File: static/exchange.html [hunk 1]
```diff
@@ -109,12 +109,54 @@ <h3>Publish message</h3>
...
+              <span class="tooltiptext">Message identifier of the message used by application.</span>
+            </a> |
+            <a id="property-tag" class="arg-tooltip" data-tag="timestamp">Timestamp
+              <span class="tooltiptext">Timestamp of the message used by application.</span>
+            </a> |
+            <a id="property-tag" class="arg-tooltip" data-tag="type">Type
+              <span class="tooltiptext">Type of the message used by application.</span>
+            </a> |
+            <a id="property-tag" class="arg-tooltip" data-tag="user_id">User ID
+              <span class="tooltiptext">User identifier when impersonate another user.</span>
```
Comment (viktorerlingsson) [inline static/exchange.html:136]: when **impersonating** sounds better IMO

**Review by viktorerlingsson — APPROVED** [review]
LGTM!

File: static/main.css [hunk 2]
```diff
@@ -30,6 +30,7 @@ textarea {
 
 a:hover {
   cursor: pointer;
+  color:#6a5855;
```
Comment (carlhoerberg) [inline static/main.css:33]: missing space between key and value

File: static/main.css [hunk 3]
```diff
@@ -925,9 +926,15 @@ pre.arguments > div {
...
   text-indent: -1em;
 }
+
 #dataTags {
   margin-left :150px;
   padding: 7px;
+  max-width: 400px;
+}
+
+#property{
```
Comment (carlhoerberg) [inline static/main.css:936]: missing space between name and opening bracket

**Pull request merged**